crates/astria-core/src/sequencerblock/v1/block.rs

```diff
@@ -1042,6 +1045,16 @@ fn take_initial_elements_from_data(
          `data` used to construct the tree",
     );
 
+    let extended_commit_info = if with_extended_commit_info {
```

Comment:
This assigned variable is never used.

Reply:
it needs to be taken out of the iter either way, and next PR i will be using this value in `SequencerBlock`. should i still remove from here?

Reply:
I think this is fine to leave as is since this is being merged to a feature branch rather than `main`.